core/src/main/scala/kafka/cluster/Broker.scala

```diff
@@ -79,17 +80,21 @@ object Broker {
             else if (version == 1) {
               val host = brokerInfo("host").asInstanceOf[String]
               val port = brokerInfo("port").asInstanceOf[Int]
-              Map(SecurityProtocol.PLAINTEXT -> new EndPoint(host, port, SecurityProtocol.PLAINTEXT))
+              val securityProtocol = SecurityProtocol.PLAINTEXT
+              val endPoint = new EndPoint(host, port, ListenerName.forSecurityProtocol(securityProtocol), securityProtocol)
+              Seq(endPoint)
             }
             else {
+              val securityProtocolMap = brokerInfo.get("listener_security_protocol_map").map(
```

Comment:
Yeah, I added a note to do that along with the test that verifies the typo fix. I'll change the code to avoid the second map.